python/lsst/pipe/tasks/postprocess.py

```diff
@@ -637,3 +684,105 @@ def writeMetadata(self, dataRef):
         """No metadata to write.
         """
         pass
+
+
+class TransformSourceTableConfig(TransformCatalogBaseConfig):
+    pass
+
+
+class TransformSourceTableTask(TransformCatalogBaseTask):
+    """Transform/standardize a Source catalog
+    """
+    _DefaultName = "transformSourceTable"
+    ConfigClass = TransformSourceTableConfig
+
+    inputDataset = 'source'
+    outputDataset = 'sourceTable'
+
+    def writeMetadata(self, dataRef):
+        """No metadata to write.
+        """
+        pass
+
+    def writeConfig(self, butler, clobber=False, doBackup=True):
+        """No config to write.
```

Comment:
There is a config to write for this task, right?

Reply:
(There's no config now... but you might have been looking at an older commit where there was)